src/Compilers/Core/Portable/MetadataReference/MetadataReferenceProperties.cs

```diff
@@ -134,6 +143,8 @@ public ImmutableArray<string> Aliases
         /// </summary>
         public bool EmbedInteropTypes => _embedInteropTypes;
 
+        internal bool IsRecursive { get; private set; }
```

Comment:
Perhaps the name should more specific? I.e. RecursivelyPropagatesAliasesToDependencies, or something like that.